### DISCUSSION TITLE
How can I model a positron emission tomography (PET) system with EGSnrc?

### DISCUSSION THREAD

**Discussion by aminetec** [nrc-cnrc/EGSnrc#808] [Questions] [answered]
I'm looking to simulate PET scanner system using EGSnrc but I can't seem to find the `egs_pet` example in `HEN_HOUSE`. Any help would be greatly appreciated.

**Comment by ftessier** [comment, marked as answer]
The `egs_pet` application has never been released with EGSnrc officially, although development versions were distributed among researchers in the field. If you are only looking to model the positron emission and scoring in the detector crystals, i.e., without modelling the electronic digitization chain, then you can build the detector using the standard egs++ geometries and sources.
